test/Concurrency/experimental_feature_strictconcurrency.swift

```diff
@@ -20,3 +20,35 @@ struct Test2: TestProtocol { // expected-warning{{conformance of 'C2' to 'Sendab
   // expected-note@-1{{in associated type 'Self.Value' (inferred as 'C2')}}
   typealias Value = C2
 }
+
+@globalActor
+actor TestGlobalActor {
+  static var shared = TestGlobalActor()
+}
+
+@TestGlobalActor
+var mutableIsolatedGlobal = 1
+
+var mutableNonisolatedGlobal = 1 // expected-error{{var 'mutableNonisolatedGlobal' is not concurrency-safe because it is global shared mutable state}}
+// expected-note@-1{{isolate 'mutableNonisolatedGlobal' to a global actor, or convert it to a constant}}
```

Comment:
Maybe it could be useful to produce a fix-it to make `var` into `let`?

Reply:
Your feedback is making me realize that the diagnostics could be improved by being more detailed, and I just completed some rewordings. I think however offering this fixit is complicated by the fact that there are two potential solutions here (either isolating to a main actor, or making a constant that is also 'Sendable', and Sensibility was something that you made me realize is missing from this note), and we don't automatically know which would be most suitable.

Reply:
Ah I see, yeah it makes sense and new diagnostic with `Sendable` bit sounds good =]